src/Sportradar.OddsFeed.SDK/Entities/REST/ICompetitor.cs

```diff
@@ -122,5 +122,11 @@ public interface ICompetitor : IPlayer
         /// </summary>
         /// <value>The short name</value>
         string ShortName => null;
+
+        /// <summary>
+        /// Ensures the profile is loaded for the competitor. This makes use of async await to avoid Task.Run().Wait() deadlocks
+        /// </summary>
+        /// <returns></returns>
+        Task EnsureProfileLoaded();
```

Comment:
This is an addition to a public interface and there are no changes that alter functionality there are no breaking changes.